tools/topology/m4/muxdemux.m4

```diff
@@ -2,6 +2,24 @@ divert(-1)
 
 dnl Define macro for demux widget
 
+define(comp_type, 0)
+define(sof_magic, 0x00464F53)
+define(stream_data_size, 16)
+define(mux_config_size, 8)
```

Comment:
8 what? unit?

Reply:
again struct size, so needs to be more clear